### DISCUSSION TITLE
Update primary email when old one no longer exists

### DISCUSSION THREAD

**Discussion by relativeflux** [community/community#57537] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,

The primary email address for my GitHub account no longer exists, as I changed my job and that email got deleted. If I change the primary email to something else will I first have to confirm my old email, or something like that? What I mean is, will a conformation step happen, where I have to go to my old primary email, or will I just have to confirm the new email I want to change it to?

Many thanks

**Comment by mrepol742** [comment]
You can add your new email then verify it and make it your primary one after it you can delete your old one.

**Reply by airtower-luna** [reply]
Or don't delete the old one, if you do any commits using it won't show as yours any more.

**Reply by relativeflux** [reply]
Ah yes, I'd read of something like that happening to people... So I guess I'll retain the old email, even though it doesn't work any more. Isn't there a way of avoiding that by getting a github email? I think I saw that suggested somewhere, unfortunately I don't have a link to it now.

**Reply by airtower-luna** [reply]
Go to your settings under Email, there you can select a "primary" and a "backup" address. Make sure the old address isn't one of those. There's also Notifications, where you can select where notification mails go.

**Comment by mwt** [comment, marked as answer]
I've done it a few times. There is no confirmation step for changing the primary email. You only need to confirm new emails when you add them.

**Reply by amy042924** [reply]
It doesn't work. I keeps sending verification code to my primary email which I can't access any more
